lib/osut/utils.rb

```diff
@@ -2834,21 +2834,25 @@ def verticalPlane(p1 = nil, p2 = nil)
   # @param pts [Set<OpenStudio::Point3d] 3D points
   # @param n [#to_i] requested number of unique points (0 returns all)
   #
-  # @return [OpenStudio::Point3dVector] unique points (see logs if empty)
+  # @return [OpenStudio::Point3dVector] unique points (see logs)
   def getUniques(pts = nil, n = 0)
     mth = "OSut::#{__callee__}"
     pts = to_p3Dv(pts)
-    ok  = n.respond_to?(:to_i)
     v   = OpenStudio::Point3dVector.new
     return v if pts.empty?
-    return mismatch("n unique points", n, Integer, mth, DBG, v) unless ok
+
+    if n.is_a?(Numeric)
```

Comment:
Safer to check for _Numeric_ than `respond_to?(:to_i)`. Non-numeric strings `respond_to?(:to_i)`, for instance.